sphinx/source/conf.py

```diff
@@ -27,7 +27,7 @@
 # The short X.Y version
 version = ''
 # The full version, including alpha/beta/rc tags
-release = '1.28.0'
+release = '1.28.1'
```

Comment:
Please do not change that! This is done during the build of the package!

Reply:
> Hello @gronastech!
> 
> 1. How can I reproduce the error?

> Best regards,
> Oliver

Such issues appear when socket restart initiated (abnormal connection closed, etc - any reason). And at the same time channel was empty - not subscription to exact trade pairs, just open depth stream.

You have typical issue across the code when you don't check that key exists in the dictionary and trying to open it directly.



The same issue appeared for !userData channel:
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/lordserenity/.local/lib/python3.8/site-packages/unicorn_binance_websocket_api/unicorn_binance_websocket_api_manager.py", line 433, in _create_stream_thread
    loop.run_until_complete(socket.start_socket())
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/lordserenity/.local/lib/python3.8/site-packages/unicorn_binance_websocket_api/unicorn_binance_websocket_api_socket.py", line 114, in start_socket
    received_stream_data = self.unicorn_fy.binance_com_websocket(received_stream_data_json)
  File "/home/lordserenity/.local/lib/python3.8/site-packages/unicorn_fy/unicorn_fy.py", line 85, in binance_com_websocket
    return UnicornFy.binance_websocket(stream_data_json, exchange="binance.com", show_deprecated_warning=False)
  File "/home/lordserenity/.local/lib/python3.8/site-packages/unicorn_fy/unicorn_fy.py", line 255, in binance_websocket
    if stream_data['data']['e'] == 'aggTrade':
KeyError: 'data'

Reply:
thx